### DISCUSSION TITLE
Citi Hackathon FDC3 Use Case: Upgrade FDC3 website to docusaurus >=3.5.2

### DISCUSSION THREAD

**Discussion by sivaprasadanakarla** [finos/finos-pmcs#334] [Ideas]
Forked and started working on this migration issue

**Comment by VijaysSriya** [comment]
> Forked and started working on this migration issue

Hi , would like to work together and get this done

**Reply by Tanisha0708** [reply]
Yessss

**Comment by kriswest** [comment]
@sivaprasadanakarla @VijaysSriya and @DILPREET1910, great to see you all taking on this problem. I (and at least one other FDC3 maintainer) have posted some feedback on the two PRs received so far:

- https://github.com/finos/FDC3/pull/1406
- https://github.com/finos/FDC3/pull/1409

There are merits to both PRs, but both need some additional work. See the comments for details.
